tools/GraphClust/CollectResults/glob_report.xml

```diff
@@ -73,7 +73,7 @@
         #end if
 
         &&
-        python '$__tool_directory__/evaluation.py'
+        python '$__tool_directory__/evaluation.py' FASTA/ RESULTS/
```

Comment:
I'm wondering why you don't need to update your tests ;)
Maybe you should add a test for this?

Reply:
Functionality is the same :) Before the folders were hard coded to be `FASTA/ and RESULTS/` . Now it's passed to the script, so I can use same script for local tests, with custom directory names.

Reply:
Hmm the output is tested here, but stays the same :)

Reply:
RESULTS/evaluation.txt